templates/taxi/car_detail.html

```diff
@@ -1,22 +1,35 @@
 {% extends "base.html" %}
-
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <div class="d-flex justify-content-between align-items-center">
+    <h1>Drivers</h1>
+    <div>
+      {% if request.user in car.drivers.all %}
+        <form method="post" action="{% url 'taxi:car-remove-driver' pk=car.id %}">
+          {% csrf_token %}
+          <input type="submit" value="Delete me from this car" class="btn btn-danger">
+        </form>
+      {% else %}
+        <form method="post" action="{% url 'taxi:car-assign-driver' pk=car.id %}">
+          {% csrf_token %}
+          <input type="submit" value="Assign me to this car" class="btn btn-success">
+        </form>
+      {% endif %}
```

Comment:
Ensure that the logic for assigning or removing the current user as a driver aligns with the application's intended functionality. Verify that the user has the appropriate role or permissions to perform these actions.